src/less/avatar/avatar.less

```diff
@@ -0,0 +1,111 @@
+@import "../variables/variables.less";
+
+.avatar {
```

Comment:
The letters don't look to be vertically centered. 
<img width="336" alt="image" src="https://user-images.githubusercontent.com/1675667/236001256-4ca5c6b4-4cbb-4c3d-a41c-cdf7350d80d5.png">